cbor-schema/plt.cddl

```diff
@@ -129,12 +130,19 @@ token-list-update-details = {
     "target": token-holder
 }
 
+; Pause or unpause the execution of certain token operations. This includes
+; the operations "mint", "burn", and "transfer".
+token-pause = {
+    ; Whether to pause or unpause token operations.
+    "pause": bool
```

Comment:
I think the body of all operations should be a map. I would suggest having separate `pause` and `unpause` operations with the empty map as the body.

Reply:
🙈 

I had a feeling you'd say that.. I'll draw up another proposal, and we can start a discussion there.